### ISSUE TITLE
"About/help" button does not show version number

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#118]
```
Click the About/help button from the main menu.

What is the expected output? 
Common custom dictates that any "About..." feature should contain a version 
number somewhere.

What do you see instead?
No version number. No help either, for that matter, but that's another issue.

What version of the product are you using? 
Well HOW SHOULD I KNOW?

(sorry. unable to resist that one. =)
```

Original issue reported on code.google.com by `Hoh...@gmail.com` on 3 Jan 2011 at 12:36

**Comment by GoogleCodeExporter** [comment]
```
Similar to issue 110.
```

Original comment by `kims...@gmail.com` on 3 Jan 2011 at 1:50

**Comment by GoogleCodeExporter** [comment]
```
...but not quite the same. =)

Anyway, I've more or less kept the app running since I installed it, so the 
current solution of having the version number show only during startup (which I 
totally had forgotten all about) was of very little help to me. =)
```

Original comment by `Hoh...@gmail.com` on 3 Jan 2011 at 1:59

**Comment by GoogleCodeExporter** [comment]
```
Why ?
Its the same location you ONLY can see the "about/help" button.
```

Original comment by `kims...@gmail.com` on 3 Jan 2011 at 11:00

**Comment by GoogleCodeExporter** [comment]
```
Well, look at that. There it is in the upper right corner.

D'oh.

To my defense, I was tired when filing the issue after having played non-stop 
all night. =)

Also, I must say that the light grey font color of the version number was very 
well camouflaged by the slightly less light grey background.
```

Original comment by `Hoh...@gmail.com` on 3 Jan 2011 at 1:39

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. I'm happy to hear you enjoy the game.

I agree that "About" should contain the version number somewhere. I'm adding it 
to that screen.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jan 2011 at 8:17
- Added labels: **Milestone-0.6.8**

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 9 Jan 2011 at 10:51
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.8 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 9:34
- Changed state: **Fixed**
